tasks/utils.py

```diff
@@ -168,14 +174,13 @@ def get_git_commit():
     """
     return check_output(['git', 'rev-parse', '--short', 'HEAD']).decode('utf-8').strip()
 
-def get_default_python():
+def get_default_python(python_runtimes):
     """
     Get the default python for the current build:
-    - default to 3 if PYTHON_RUNTIMES is not specified (so that dev builds default to 3)
-    - default to 2 if PYTHON_RUNTIMES includes both versions (so that builds with 2 and 3 default to 2)
+    - default to 2 if python_runtimes includes 2 (so that builds with 2 and 3 default to 2)
+    - default to 3 if python_runtimes includes only 3
```

Comment:
nit: in actuality it will default to '3'  if '2' not in `python_runtimes`, no matter what it is.